src/components/Pages/RegisterPage.tsx

```diff
@@ -51,17 +54,91 @@ const Button = styled.button`
 `;
 
 const RegisterPage = () => {
+	const [values, setValues] = useState({
+		firstname: '',
+		lastname: '',
+		username: '',
+		email: '',
+		password: '',
+		confirmPassword: '',
+	});
+
+	const inputs = [
```

Comment:
extract outside of component, as right now this const is redefined on each render